src/runtime/manifest-primitive-parser/smoke/toy.html

```diff
@@ -79,7 +80,7 @@
         `Stores: ${ManifestPrimitiveParser.extract('store', ast).length}`
       ].join('\n');
     } catch(x) {
-      logElt.innerText = x.toString();
+      logElt.innerHTML = `<span style="color:darkred;">${x.toString()}</span>`;
```

Comment:
Out of curiosity, why did you use this approach instead of the classList? (I'm not objecting, just interested as to why.)

Reply:
Simplicity